source/client/cl_parse.c

```diff
@@ -1217,10 +1218,28 @@ static void CL_ParseConfigstringCommand( void )
 		idx = atoi( Cmd_Argv( i ) );
 		s = Cmd_Argv( i + 1 );
 
+
 		CL_UpdateConfigString( idx, s );
 	}
 }
 
+static void CL_SteamAuth(){
+	if (Steam_Active())
+	{
+		// ticket needs to be generated on demand each time we join a server
+		SteamAuthTicket_t *ticket = Steam_GetAuthSessionTicketBlocking();
+
+	  uint8_t messageData[MAX_MSGLEN];
+		msg_t msg;
+		MSG_Init(&msg, messageData, sizeof(messageData));
+	  MSG_WriteByte(&msg, clc_steamauth);
+	  MSG_WriteLong(&msg, ticket->pcbTicket);
+		MSG_WriteData(&msg, ticket->pTicket, AUTH_TICKET_MAXSIZE);
```

Comment:
Mixed indenting here (and probably in more places), I'd suggest your editor to use the `.editorconfig` file.

Reply:
what editorconfig? there doesn't seem to be one in the repository. I did try using the .clang-format and uncrustify but both changed existing conventions